03-talos.tf

```diff
@@ -10,6 +10,8 @@ module "talos_control_plane_nodes" {
   subnet_id                   = element(data.aws_subnets.public.ids, count.index)
   associate_public_ip_address = true
   tags                        = merge(var.tags, local.cluster_required_tags)
+  metadata_options            = var.metadata_options
```

Comment:
Remove this change, as it's already covered in your other PR: https://github.com/isovalent/terraform-aws-talos/pull/171